### ISSUE TITLE
url dispatcher incompatible with django 1.0

### ISSUE THREAD

**Issue by vijaykramesh** [django-tastypie/django-tastypie#56]
So first of all, I'm not even sure whether tastypie is supposed to work with Django 1.0 or not.  It claims >= 1.0 in the main documentation (http://toastdriven.github.com/django-tastypie/tutorial.html) but 1.1 at the github page (https://github.com/toastdriven/django-tastypie). With that said, I found an issue with the way tastypie is doing url dispatching that prevents it from (easily) working with Django 1.0 - namely, it wasn't until 1.1 that include() was extended to "accept as an argument an interable that returns URL patters" (compare http://docs.djangoproject.com/en/1.0/topics/http/urls/#include to http://docs.djangoproject.com/en/1.1/topics/http/urls/#include).

I am very impressed with tastypie, but am stuck working in Django 1.0 for a large project; it would be easy enough to either hack tastypie to have a Django 1.0 compatible my_resource.urls_1 function but I don't want to commit myself to using tastypie if other issues might arise in Django 1.0.

Basically, I guess my question is - has anybody successfully used tastypie in Django 1.0?  And if so, did you have to modify anything else other than the default way it is doing url dispatching?

Thanks.
